.vib/external-dns/goss/goss.yaml

```diff
@@ -0,0 +1,11 @@
+file:
+  /opt/bitnami/external-dns/bin:
+    mode: "0755"
+    filetype: directory
+    exists: true
+command:
+  user-id-test:
```

Comment:
I'd add another extra test to check that the version matches:

```
$ external-dns --version
v0.12.0
```

Reply:
Also, you may check that port `7979` (metrics) is in use 😁 WDYT?

Reply:
Added both of those tests!